index.js

```diff
@@ -48,20 +49,41 @@ async function run() {
                         return;
                     }
 
-                    let file_content = readFileSync(`${template_dir}${filename}`, {encoding: "utf8"});
+                    create_comment(octokit, issue, template_dir, filename);
 
-                    octokit.issues.createComment({
-                        owner: issue.owner,
-                        repo: issue.repo,
-                        issue_number: issue.number,
-                        body: file_content,
-                    });
+                    return;
                 }
             });
         });
+
+        // No pattern matched, let's check for the default template
+        if (default_template) {
+            let filename = `${default_template}.md`;
+
+            if (!templates.includes(filename)) {
+                core.setFailed(`Could not find template: ${filename}!`);
+                return;
+            }
+
+            create_comment(octokit, issue, template_dir, filename);
+        }
     } catch (error) {
         core.setFailed(error.message);
     }
 }
 
+/**
+ * Post a comment with the content of `filename` to a given GitHub issue.
+ */
+function create_comment(octokit, issue, template_dir, filename) {
```

Comment:
Why do you need to pass octokit in the function, it looks like it's declared globally ?

Reply:
disregard je l'ai manqué en haut